### ISSUE TITLE
Image preview size do not change with gallery size

### ISSUE THREAD

**Issue by ladailam382** [gradio-app/gradio#4850]
### Describe the bug

In my project, I used gr.Gallery to display images and I changed the height of the gallery. However, when I click on an image to open preview mode, the height of preview image is not the same as that of the gallery. Is there any suggestion for this situations?

Here is what my screen look like when I click on an image. Thank you in advance!

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Here is the code:
`with gr.Row():
            output = gr.Gallery(value=['image/example/butterfly.png', 'image/example/1.png']).style(grid=[2], height="600px")
`

### Screenshot

![image](https://github.com/gradio-app/gradio/assets/75711891/e4b0ac57-7ee6-4cb2-a783-40a8e1e2cae3)

### System Info

```shell
I'm using windows 11 with following gradio package:

Name: gradio
Version: 3.30.0
```

### Severity

Blocking usage of gradio

**Comment by hysts** [comment]
Hi @ladailam382 
I think it's fixed in `gradio==3.36.1`. (See https://github.com/gradio-app/gradio/issues/4542)
As `.style` is deprecated in `gradio==3.36.1`, you can do the following:
```python
with gr.Row():
    output = gr.Gallery(value=['image/example/butterfly.png', 'image/example/1.png'], columns=2, height="600px")
```

**Comment by ladailam382** [comment]
> Hi @ladailam382 I think it's fixed in `gradio==3.36.1`. (See #4542) As `.style` is deprecated in `gradio==3.36.1`, you can do the following:
> 
> ```python
> with gr.Row():
>     output = gr.Gallery(value=['image/example/butterfly.png', 'image/example/1.png'], columns=2, height="600px")
> ```

Thank you for your quick response, I just change the version as well as code and it works!!!
